golang/cosmos/x/swingset/swingset.go

```diff
@@ -17,6 +19,14 @@ type swingsetMessage struct {
 	Request json.RawMessage `json:"request"`
 }
 
+type swingsetStoreSetRequest struct {
```

Comment:
Will the JSON of `swingsetMessage` have `"request"` fields of different types? If the `"request"` field is always a `swingsetStoreSetRequest`, then you can avoid the `json.RawMessage` indirection. See how `x/vbank` unmarshals all operations into a structure with the union of all fields.